wpilibc/src/main/native/cpp/AnalogGyro.cpp

```diff
@@ -64,6 +65,18 @@ AnalogGyro::AnalogGyro(std::shared_ptr<AnalogInput> channel, int center,
 
 AnalogGyro::~AnalogGyro() { HAL_FreeAnalogGyro(m_gyroHandle); }
 
+AnalogGyro::AnalogGyro(AnalogGyro&& rhs) {
+  m_analog = std::move(rhs.m_analog);
+  std::swap(m_gyroHandle, rhs.m_gyroHandle);
+}
```

Comment:
the optimizer will probably fix this, but is there a reason you aren't doing a member initializer list, at least for the parts you're calling std::move for?

Reply:
It was consistent with what I was doing last night and I was fighting the clock to get to bed at a reasonable hour. :)